### DISCUSSION TITLE
I dont know how to use this

### DISCUSSION THREAD

**Discussion by Shikomi36** [gekigek99/minecraft-vanilla-server-hibernation#142] [Help required] [answered]
I have no idea on how to even start using this, could i get a clearer explanation?

**Comment by gekigek99** [comment, marked as answer]
is your minecraft server already installed and you can access it?

**Reply by Shikomi36** [reply]
yes, and on a side note your discord link is expired.
would it be easier for you to talk through discord?

**Reply by gekigek99** [reply]
(updated it thanks for reporting)

**Reply by gekigek99** [reply]
1) just download the lastest executable for your platform + config.json and put them in the minecraft server folder.
2) then edit `Folder` to `.` and `filename` to `server.jar` (this should make it work, you can play with parameters later on).
3) now edit your minecraft client to connect to `yourserveraddress.com:**25555**` instead of 25565.

**Reply by Shikomi36** [reply]
i got it to work but when it gets to the start region it does nothing

**Reply by Shikomi36** [reply]
nevermind it was a conflict with spongeforge, it works now thank you

**Reply by gekigek99** [reply]
Glad to hear it!

Did you manage to join the discord server?
If you like this repo give it a star!
